src/unit_tests/wrappers/wazuh/syscheckd/create_db_wrappers.c

```diff
@@ -64,3 +64,21 @@ int __wrap_fim_whodata_event(whodata_evt * w_evt)
 #endif
     return 1;
 }
+
+/**
+ * @brief This function load the expect and will_return calls for the wrapper of fim_checker
+*/
+void expect_fim_checker_call(const char *path, int w_evt, int report) {
+    expect_string(__wrap_fim_checker, path, path);
+    expect_value(__wrap_fim_checker, w_evt, w_evt);
+    expect_value(__wrap_fim_checker, report, report);
+}
+
+/**
+ * @brief This function load the expect and will_return calls for the wrapper of fim_configuration_directory
```

Comment:
This function **loads**. Also, this doxygen comment should be in the _.h_ file instead.